charts/camunda-platform/templates/tasklist/deployment.yaml

```diff
@@ -75,6 +75,13 @@ spec:
               {{- end }}
             - name: CAMUNDA_TASKLIST_IDENTITY_AUDIENCE
               value: "tasklist-api"
+            {{- if .Values.tasklist.identityUserAccessRestrictionsEnabled }}
```

Comment:
I'd first suggest using a different key name here.

How is the var `CAMUNDA_TASKLIST_IDENTITY_USER_ACCESS_RESTRICTIONS_ENABLED` represented in Tasklist? Is it `camunda.tasklist.identity.userAccessRestrictions.enabled` or something else?

Reply:
Ok :) I understood now, gonna update the identifier

Reply:
This one is good `tasklist.identity.userAccessRestrictions.enabled` because we can extend both `identity` and `userAccessRestrictions` if needed.

But the current implementation looks like `tasklist.identityUserAccessRestrictions.enabled` (the `identity` is not a map).